package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "tsc": "tsc",
     "watch-build": "npx tsc --watch",
-    "lint": "eslint --ext .ts,.tsx ./",
+    "lint": "eslint --ext .ts ./ --fix",
```

Comment:
what does "fix" do? 
do you REALLY REALLY want it here :D ?

Reply:
my point - you should solve the underlying problem - this masks the problem and does something else - automatically modifies code you don't even commit. 
won't this modify the code and fix it - but you're not commiting that.

look at editorconfig. 
https://editorconfig.org/